bps/templates/bps_verify_bfk.yaml

```diff
@@ -0,0 +1,11 @@
+pipelineYaml: "${CP_VERIFY_DIR}/pipelines/${INSTRUMENT}/verifyBfk.yaml"
+
+project: "${TICKET}"
+campaign: "${TICKET}"
+
+payload:
+  payloadName: "${INSTRUMENT}_${TICKET}"
+  output: "${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET}/${TAG}/verifyBfk.${VERIFY_RERUN}"
+  butlerConfig: "${REPO}"
+  inCollection: "${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET}/${TAG},${RAW_COLLECTION},${CALIB_COLLECTIONS}"
```

Comment:
It didn't occur to me until I saw this here, but the README should note that the inputs need to be certified and chained onto the top level `${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET}/${TAG}` collection before the next step is done.